drivers/sensor/lm75/lm75.c

```diff
@@ -83,6 +84,11 @@ static int lm75_sample_fetch(const struct device *dev,
 		break;
 	}
 
```

Comment:
You probably want to check for `ret` here and return the value if negative, that way you can reuse the same variable.

Reply:
@CodingGhost . Hmm indeed after rereading the patch, you are indeed correct. Then I don't see any other choice than using two variables to store the return codes.